RULES.md

```diff
@@ -223,5 +223,118 @@ your code.
 * Always use semicolons.
 
   ```js
+<<<<<<< HEAD
```

Comment:
another merge conflict. seems pretty easy to resolve, just remove the HEAD version.